src/python/pants/option/global_options.py

```diff
@@ -781,15 +781,18 @@ def register_bootstrap_options(cls, register):
         register(
             "--pantsd-max-memory-usage",
             advanced=True,
-            type=int,
-            default=2 ** 30,
+            type=memory_size,
+            default="1GiB",
```

Comment:
FYI `./pants help` will show this normalized as an int of bytes. I wish it showed "1GiB", but such it is. So, I added a brief explanation of the default to help message.

Reply:
We could in the future add a new option type for this.

Reply:
We could have the help system understand the custom type. We'll want to do something similar for `--process-execution-local-parallelism`, as you pointed out in another context.